src/FlameGraph.js

```diff
@@ -12,6 +12,7 @@ type Props = {|
   data: ChartData,
   height: number,
   width: number,
+  onFocus?: (chartNode: ChartNode) => void,
```

Comment:
I'm not sure this is the best name. "onFocus" sounds to me like the flame graph itself has been focused/blurred. Maybe "onChange"?